app/i18n/cz/admin.php

```diff
@@ -191,6 +191,7 @@
 		'feed_count' => 'Feeds',	// TODO - Translation
 		'language' => 'Jazyk',
 		'list' => 'User list',	// TODO - Translation
+		'mail' => 'Email',	// TODO - Translation
```

Comment:
I would use `email` instead of `mail` to be coherent with what I use in the email validation feature. Renaming `mail_login` in `email` is one of the task I plan to do later (but I have to do some other things before ^^)

Reply:
Sure!